server/src/Models/ClinicalDetails/Investigation.ts

```diff
@@ -1,7 +1,6 @@
 interface Investigation {
     isolationStartTime: Date | null;
```

Comment:
You need to add isInIsolation and does have a symptoms.

Reply:
fixed